MAUI/Cartesian-Charts/RangeArea.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Range Area Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about range area chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Range Area Chart in .NET MAUI Chart
+
+## Range Area Chart
+
+Range Area Chart is a type of data visualization useful for displaying the relationship between two variables over time. In this Series, the area between two lines is filled to indicate a range of values, such as high and low price range or an upper and lower limit.
+
+By displaying ranges of data, range area series can make it easier to compare multiple datasets at once.
+
+To render a range area chart, create an instance of [RangeAreaSeries](), and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
```

Comment:
To render a range area chart, create an instance of the [RangeAreaSeries](), and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).